### PR TITLE
Add lens-extension-certificate-info

### PR DESCRIPTION
**Pull request by jkroepke**
Hi,

it's my first extension for lens based on https://github.com/lensapp/lens/issues/2632

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ You'll find list of publicly available [Lens](https://k8slens.dev) extensions in
 - [Capsule](https://github.com/clastix/capsule-lens-extension) by [Clastix](https://clastix.io/)
 - [Carbonetes Container Security](https://github.com/carbonetes/carbonetes-lens-extension) by [Carbonetes](https://carbonetes.com/)
 
+## Informational
```
Comment (jim-docker) [inline README.md]:
```suggestion
## Community Extensions
```

**Comment by jkroepke** [comment]
@jim-docker Suggestions applied

**Comment by jim-docker** [comment]
Hi @ jkroepke, thanks for this, it's a nice little extension, and you've gone to the extra effort of making it accessible for everyone. We are holding off on this PR because we will be releasing Lens v5 soon and things around extensions are changing quite a bit (there are breaking changes, extensions written for v4 are incompatible with v5 and will require modifications). We'll be updating the [extension api docs](https://api-docs.k8slens.dev/master/extensions/api/) soon. We hope you will bring it up to date for v5.

**Comment by jkroepke** [comment]
Are you going to remove all existing extensions from the README.md, too? I would expect they are not compatible with lens 5, too.

Otherwise, you can merge this PR +
I add a compatibility matrix that the master isnt compatible with lens 5 yet.

**Review by jim-docker — APPROVED** [review]
This may be reorganized as part of Lens 5 changes, but merging anyway to have a record of this extension

**Pull request merged**